include/ocpp/v201/messages/PublishFirmware.hpp

```diff
@@ -17,7 +17,7 @@ namespace v201 {
 
 /// \brief Contains a OCPP PublishFirmware message
 struct PublishFirmwareRequest : public ocpp::Message {
-    CiString<512> location;
+    CiString<2000> location;
```

Comment:
Adding a `FieldLength` variable for `PublishFirmwareRequest.location` should make this backwards-compatible with 2.0.1